bl-languages/fi_FI.json

```diff
@@ -0,0 +1,242 @@
+{
+	"language-data":
+	{
+		"native": "Suomi (Suomi)",
+		"english-name": "Finnish",
+		"last-update": "2016-06-11",
+		"author": "Tuomas K.",
+		"email": "",
+		"website": ""
+	},
+
+	"username": "Käyttäjätunnus",
+	"password": "Salasana",
+	"confirm-password": "Vahvista salasana",
+	"editor": "Kirjoittaja",
+	"dashboard": "Hallintapaneeli",
+	"role": "Tyyppi",
+	"post": "Blogikirjoitus",
```

Comment:
"post": "Julkaisu",
(Julkaisu is a slightly more vague term but a shorter word and fits the context.)